Workbooks/Virtual Machines/Availability/Server Availability.workbook

```diff
@@ -0,0 +1,282 @@
+{

```

Comment:
why does this show up as a new file?  aren't you modifying an existing template?

Reply:
That's right I edited the existing file, i'm not sure why it's showing as new

Reply:
did you modify it in an old branch and now you're merging that branch again?

because this branch somehow seems to have commits going back 4 years?

![image](https://user-images.githubusercontent.com/10158007/218875658-e3c1daf1-d78c-4d2d-8649-4a5876028b16.png)

i'd start over, pull from master, create a new branch from master, and then modify the template?